src/stories/sponsors_tier.tsx

```diff
@@ -0,0 +1,233 @@
+        break;
+      default:
+        setCurrentContentData(goldContentData);
+    }
+
+    // Reset currentContent to the first item in the new content data
+    setCurrentContent(0);
+    setTransitionDirection("newCard");
+  };
+
+  const { image, header, subheader, text, link } =
+    currentContentData[currentContent];
+
+  return (
+    <div
+      className={`flex mobile-response flex-col bg-white border rounded-xl shadow-sm dark:bg-slate-900 dark:border-gray-700 dark:shadow-slate-700/[.7]`}
+    >
+      {/* DNA Pic */}
+      <div className="flex-shrink-0 relative w-full rounded-t-xl overflow-hidden pt-[40%] sm:rounded-s-xl sm:max-w-[15rem] md:rounded-se-none md:max-w-xs">
+        <div className="dna-pic">
+          {/* Navigation buttons for different sponsor types */}
+          <button
+            className="content-button text-white"
+            onClick={() => handleContentChange("gold")}
+          >
+            Gold Sponsors
+          </button>
+          <button
+            className="content-button text-white"
+            onClick={() => handleContentChange("silver")}
+          >
+            Silver Sponsors
+          </button>
+          <button
+            className="content-button text-white"
+            onClick={() => handleContentChange("bronze")}
+          >
+            Bronze Sponsors
+          </button>
+        </div>
+      </div>
+
+      {/* Right Content */}
+      <div className={`flex flex-wrap w-full px-10`}>
+        <div
+          className={`p-4 flex flex-col h-full w-full sm:p-7  ${
+            transitionDirection === "left" ? "cardTransitionLeft" : ""
+          }
+    ${transitionDirection === "right" ? "cardTransitionRight" : ""}
+    ${transitionDirection === "newCard" ? "newCardTransition" : ""}`}
+          onAnimationEnd={handleTransitionEnd}
+        >
+          <div className="flex flex-row items-center">
+            <div className="logo-temp"></div>
+            <h2 className="text-2xl font-bold">{header}</h2>
+          </div>
+          <h3 className="text-lg subheader">{subheader}</h3>
+          <p>{text}</p>
+          <a
+            className="link explore-more"
+            href={link}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Explore More
+          </a>
+
+          {/* Navigation Bar */}
+          <div className="mt-5 sm:mt-auto flex justify-center">
+            <button
+              className="p-2 nav-button"
+              onClick={() => handleNavigation("prev")}
+            >
+              &lt;
+            </button>
+            {/* Map through contentData to create image icons */}
+            {currentContentData.map((content, index) => (
+              <button
+                key={index}
+                className={`w-8 h-8 rounded-full contentDataPlaceholder ${
+                  currentContent === index
+                    ? "border-2 border-blue-500 green"
```

Comment:
Forgot to change this when I merged in the global.css fix, but now that the `green` class is gone just use a regular tailwind class to set the button background - ex. `bg-green-500`